dataplicity/m2m/wsclient.py

```diff
@@ -363,7 +383,11 @@ def send(self, packet, *args, **kwargs):
     def send_bytes(self, packet_bytes):
         """Send bytes over the websocket."""
         with self.write_lock:
-            self.app.send(packet_bytes, binary=True)
+            if self.app.client_terminated:
```

Comment:
This avoids an exception in the log if a thread attempts to send after the connection was closed.